lib/geolookup/usa/city.rb

```diff
@@ -0,0 +1,17 @@
+module Geolookup
+  module USA
+    module City
+
+      CITY_MAP = 'CITIES.yml'
```

Comment:
why does the yml config have a city field inside each - seems repetitive since you are using the city name already as the lookup

Reply:
i will update this to omit the city: key val from the hash, thanks!